Core/include/Acts/Propagator/MultiStepperAborters.hpp

```diff
@@ -52,6 +58,22 @@ struct MultiStepperSurfaceReached {
       }
     }
 
+    // However, if mean of all is on surface, we are happy as well
+    if (averageOnSurface) {
+      const auto sIntersection = targetSurface.intersect(
+          state.geoContext, stepper.position(state.stepping),
+          state.stepping.navDir * stepper.direction(state.stepping), true);
+
+      if (sIntersection.intersection.status ==
+              Intersection3D::Status::onSurface or
+          sIntersection.intersection.pathLength < 0.2) {
```

Comment:
would it make sense to make this cut configurable?

Reply:
yeah we should probably... Actually I should try what happens if I remove this cut...